windows_containers/windows-containers-release-notes-8-x.adoc

```diff
@@ -0,0 +1,59 @@
+
+The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+ifndef::openshift-origin[]
+[id="getting-support"]
+== Getting support
+
+// wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows
+
+Windows Container Support for Red Hat OpenShift is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
+
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
+
+For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
+
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
+endif::openshift-origin[]
+
+[id="wmco-8-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 8.0.0
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 8.0.0 were released in link:https://access.redhat.com/errata/RHSA-2023:1372[RHSA-2023:1372].
+
+[id="wmco-8-0-0-new-features"]
+=== New features and improvements
+
+[id="wmco-8-0-0-os"]
+==== Support for the pod `os` parameter
+
+You can now use the `spec.os.name.windows` parameter in your workload pods to authoritatively identify the pod operating system for validation and to enforce Windows-specific pod security context constraints (SCCs). It is recommended that you configure this parameter in your workload pods. 
+
+For more information, see xref:../windows_containers/scheduling-windows-workloads.adoc#sample-windows-workload-deployment_scheduling-windows-workloads[Sample Windows container workload deployment].
+
+[id="wmco-8-0-0-must-gather"]
+==== WICD logs are added to must-gather
+
+The must-gather tool now collects the service logs generated by the Windows Instance Config Daemon (WICD) from Windows nodes.
```

Comment:
I see must-gather is put in `` in our documents. Eg. [here](https://docs.openshift.com/container-platform/4.12/virt/logging_events_monitoring/virt-collecting-virt-data.html) and [here](https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-alert-eval_understanding-and-managing-pod-security-admission)
```suggestion
The `must-gather` tool now collects the service logs generated by the Windows Instance Config Daemon (WICD) from Windows nodes.
```